player/lua/select.lua

```diff
@@ -427,3 +427,29 @@ mp.add_key_binding(nil, "show-properties", function ()
         end,
     })
 end)
+
+mp.add_key_binding(nil, "menu", function ()
+    local menu = {
```

Comment:
Shouldn't this list all select? Especially why vid select is hidden?

Reply:
Because I saw uosc doesn't have it so I didn't if it's worth adding.

Reply:
> Because I saw uosc doesn't have it so I didn't if it's worth adding.

Of course they has it. In UOSC each button is dynamic and show only when needed. See https://github.com/tomasklaen/uosc/blob/6a1c0e9c6e7e9e43cda1c1ea3e44a911fae45927/src/uosc.conf#L83C1-L83C211 specifically, `<has_many_video>video`.

This menu could do this too, to not show items that goes nowhere.